.github/workflows/pr-version-comment.yml

```diff
@@ -0,0 +1,47 @@
+name: Comment App Version on Merged PR
+
+on:
+  pull_request:
+    types:
+      - closed
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.10.16]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Verify Python version
+        run: python --version
+
+  comment-version:
```

Comment:
We can just keep everything under `comment-version`.

Reply:
We don't need `build` unless `peter-evans/create-or-update-comment` depends on some Python packages.